front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -127,7 +129,19 @@ export function InstructionScreen({
   instructionsError: string | null;
 }) {
   const editor = useEditor({
-    extensions: [Document, Text, Paragraph],
+    extensions: [
+      Document,
+      Text,
+      Paragraph,
+      History,
+      HardBreak.extend({
```

Comment:
Let's extract those extensions so we can re-use them in different places.

Reply:
@flvndvd how about now?